<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>toolbar_lucasExperiment</class>
 <widget class="QWidget" name="toolbar_lucasExperiment">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>454</width>
    <height>162</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="1" column="1">
    <widget class="QToolButton" name="multipleSeeds">
     <property name="text">
      <string>multipleSeeds</string>
     </property>
     <property name="icon">
      <iconset resource="../../../openscantools.qrc">
       <normaloff>:/icons/100x100/diameter.png</normaloff>:/icons/100x100/diameter.png</iconset>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextBesideIcon</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QToolButton" name="horizontalPlane">
     <property name="text">
      <string>horizontalPlane</string>
     </property>
     <property name="icon">
      <iconset resource="../../../openscantools.qrc">
       <normaloff>:/icons/100x100/column tilt.png</normaloff>:/icons/100x100/column tilt.png</iconset>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextBesideIcon</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QToolButton" name="planeConnexion">
     <property name="text">
      <string>planeConnexion</string>
     </property>
     <property name="icon">
      <iconset resource="../../../openscantools.qrc">
       <normaloff>:/icons/100x100/measure_points.png</normaloff>:/icons/100x100/measure_points.png</iconset>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextBesideIcon</enum>
     </property>
     <property name="autoRaise">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QToolButton" name="localPlane">
     <property name="text">
      <string>localPlane</string>
     </property>
     <property name="icon">
      <iconset resource="../../../openscantools.qrc">
       <normaloff>:/icons/100x100/measure_points.png</normaloff>:/icons/100x100/measure_points.png</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextBesideIcon</enum>
     </property>
     <property name="autoRaise">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" rowspan="2">
    <widget class="QToolButton" name="autoExtendPlane">
     <property name="text">
      <string>autoExtendPlane</string>
     </property>
     <property name="icon">
      <iconset resource="../../../openscantools.qrc">
       <normaloff>:/icons/100x100/polyline.png</normaloff>:/icons/100x100/polyline.png</iconset>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextBesideIcon</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QToolButton" name="beamBendingManual">
     <property name="text">
      <string>beamBendingManual</string>
     </property>
     <property name="icon">
      <iconset resource="../../../openscantools.qrc">
       <normaloff>:/icons/100x100/beam_bending.png</normaloff>:/icons/100x100/beam_bending.png</iconset>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextBesideIcon</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QToolButton" name="verticalPlane">
     <property name="text">
      <string>verticalPlane</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QToolButton" name="trajetoryButton">
     <property name="toolTip">
      <string>Click 2 points or 2 spheres to create the trajectory</string>
     </property>
     <property name="text">
      <string> Trajectory</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>localPlane</tabstop>
  <tabstop>autoExtendPlane</tabstop>
  <tabstop>planeConnexion</tabstop>
  <tabstop>horizontalPlane</tabstop>
  <tabstop>multipleSeeds</tabstop>
  <tabstop>beamBendingManual</tabstop>
  <tabstop>verticalPlane</tabstop>
 </tabstops>
 <resources>
  <include location="../../../openscantools.qrc"/>
 </resources>
 <connections/>
</ui>
